CHANGELOG.md

```diff
@@ -6,8 +6,12 @@
 
 ## Breaking Changes
 
+- [#632](https://github.com/oauth2-proxy/oauth2-proxy/pull/632) There is backwards compatibility to sessions from v5
+  - Any unencrypted sessions from before v5 that only contained a Username & Email will trigger a reauthentication
+
```

Comment:
Does this count as a breaking change or an important note? It's not breaking from v6.0.0 is it? Would this require a major version bump or minor?

Reply:
I'll remove the note -- it is only semi-breaking (is a reauth breaking...?) from a subset of v5 users that had a valid v6 `cookie-secret` but didn't have the 4 options that triggered a  cipher to be set. It will cause a reauthentication for those JSON sessions that come in with only an unencrypted user/email.

All other v5 use cases had options that had full sessions that would be handled with the fallback or other issues that other v6 changes forced them to re-authentication anyway